occurrences/event_api_services.py

```diff
@@ -92,32 +97,20 @@ def send_event_languages_update(
     update_event_to_linkedevents_api(p_event.linked_event_id, event_obj)
 
 
+def send_event_publish(
+    p_event: "PalvelutarjotinEvent", linked_events_data: Optional[dict]
+):
+    body = prepare_published_event_data(p_event, linked_events_data)
+    response = api_client.create("event", body)
+    response.raise_for_status()
+
+
 def send_event_republish(p_event: "PalvelutarjotinEvent"):
     # FIXME: This is a needless call if LE would not need a full event data.
     # Since the LE needs at least all the required fields when an event is updated,
     # we first need to fetch the current event object.
     event_obj = fetch_event_as_json(p_event.linked_event_id)
-
-    start_time, end_time = get_event_time_range_from_occurrences(p_event)
-    (
-        enrolment_start_time,
-        enrolment_end_time,
-    ) = get_enrollable_event_time_range_from_occurrences(p_event)
-
-    event_obj["publication_status"] = p_event.__class__.PUBLICATION_STATUS_PUBLIC
-    event_obj["start_time"] = (
-        format_linked_event_datetime(start_time) if start_time else None
```

Comment:
...`else timezone.now()` is used by the new code so there is a slight difference here. Is this difference ok in your view?